charter.md

```diff
@@ -0,0 +1,59 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus. A decision has consensus if it has support from at least two participating organizations and no opposition.
+
+The team operates in public, except when otherwise allowed by this charter.
```

Comment:
I assume the intent is that this applies transitively to the process documents defined below, but for the avoidance of doubt, we should say "by this charter, or by process documents specific to given metrics or other deliverables of the team."

Reply:
```suggestion
The team operates in public, except when otherwise allowed by this charter, or by process documents specific to given metrics or other deliverables of the team.
```